arch/arm/src/lpc17xx_40xx/lpc17_40_sdcard.c

```diff
@@ -305,9 +308,11 @@ struct lpc17_40_sampleregs_s
 static void lpc17_40_takesem(struct lpc17_40_dev_s *priv);
 #define     lpc17_40_givesem(priv) (nxsem_post(&priv->waitsem))
 static inline void lpc17_40_setclock(uint32_t clkcr);
-static void lpc17_40_configwaitints(struct lpc17_40_dev_s *priv, uint32_t waitmask,
+static void lpc17_40_configwaitints(struct lpc17_40_dev_s *priv,
+                                    uint32_t waitmask,
               sdio_eventset_t waitevents, sdio_eventset_t wkupevents);
```

Comment:
This indentation style is inconstistent with the indentation style used elsewhere (see below)